pkg/distributor/distributor_test.go

```diff
@@ -1012,12 +1012,15 @@ func TestDistributor_QueryStream_ShouldReturnErrorIfMaxChunkBytesPerQueryLimitIs
 	flagext.DefaultValues(limits)
 
 	// Prepare distributors.
+	// Use replication factor of 2 to always read all the chunks from both ingesters,
+	// this prevents us from getting chunks from a slow ingester and double counting chunks in the limiter.
```

Comment:
Genuine question, I don't know the answer and am curious :smiley:  - I'm a bit confused by the "prevents .... double counting chunks" part. With RF=2, is it not the case that chunks are _always_ counted twice?

Reply:
> With RF=2, is it not the case that chunks are always counted twice?

Yes, it is.

```suggestion
	// this guarantees us to always read the same chunks and have a stable test.
```